compatibility_server/requirements.txt

```diff
@@ -2,3 +2,4 @@ certifi
 docker
 flask
 requests
+opencensus
```

Comment:
Probably specify the version for opencensus as it is not GA yet, and newer version might introduce breaking changes, which will break our server.